src/aat/resources/features/F-032/S-126_Superuser_Case_Creation_Data_With_Document.td.json

```diff
@@ -0,0 +1,17 @@
+{
+  "_guid_": "S-126_Superuser_Case_Creation_Data_With_Document",
+  "_extends_": "F-032_Case_Creation_Data_With_Document",
+
+  "users": {
+    "invokingUser": {
+      "username": "[[$CCD_PRIVATE_CASEWORKER_EMAIL]]",
+      "password": "[[$CCD_PRIVATE_CASEWORKER_PASSWORD]]"
```

Comment:
Why does it need different user for creating a case with AAT_AUTH_1?

Reply:
we need to use a case type where one role has higher permissions than another here... I guess it is just by chance that the role with lower permissions has create permissions

Reply:
but it doesn't really matter